### PR TITLE
Added short description to remove_stale_contenttypes command.

### PR DESCRIPTION
**Pull request by touhf**
Noticed that "python manage.py help remove_stale_contenttypes" doesn't show short description like all other commands so I added one from official documentation.

### CODE REVIEW

File: django/contrib/contenttypes/management/commands/remove_stale_contenttypes.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 
 class Command(BaseCommand):
+    help = "Deletes stale content types in database."
```
Comment (felixxm) [inline django/contrib/contenttypes/management/commands/remove_stale_contenttypes.py:11]:
```suggestion
    help = "Deletes stale content types in the database."
```

**Comment by felixxm** [comment]
@touhf Please run `black` and fix linter errors.

**Comment by touhf** [comment]
@felixxm done.

**Comment by felixxm** [comment]
@touhf Thanks :+1: Welcome aboard :boat:

**Pull request merged**